pkg/apis/certmanager/validation/certificate.go

```diff
@@ -23,6 +23,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
+	"github.com/jetstack/cert-manager/pkg/util/pki"
```

Comment:
I'd like to avoid importing too many packages in anything under `apis`. Can you move the `CertDuration` method to `pkg/util/api` instead? and perhaps also rename to `(Get|Default)CertificateDuration`?